setup.py

```diff
@@ -1,11 +1,17 @@
 import glob
+import sys
 from setuptools import setup
 import unittest
 
+# For python3 wheel, we currently don't need test
+# TODO: build python3 wheel of all the dependencies, and remove conditional test
```

Comment:
Do we really plan to do this in the future?

It is my understanding that this is a temporary solution until we break the sonic Python modules out into their own package, separate from sonic-utilities, then we will build the modules package for both Python 2 and 3, correct?

Reply:
OK. This is an alternative, and we can remove TODO if either is achieved.